### PR TITLE
[stable/listmonk] Initialize Listmonk helmchart

### PR DESCRIPTION
**Pull request by javad-hajiani**
## Checklist

- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)
- [x] I have read the [contribution instructions](https://github.com/deliveryhero/helm-charts#contributing), bumped chart version and regenerated the docs
- [ ] Github actions are passing

### CODE REVIEW

File: stable/listmonk/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "listmonk.selectorLabels" . | nindent 8 }}
+        {{- if .Values.podLabels }}
+					{{ toYaml .Values.podLabels | nindent 8 }}
```
Comment (max-rocket-internet) [inline stable/listmonk/templates/deployment.yaml:23]: I think this needs some massaging

File: stable/listmonk/values.yaml [hunk 2]
```diff
@@ -0,0 +1,84 @@
+replicaCount: 2
+
+image:
+  repository: listmonk/listmonk
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  # tag: ""
```
Comment (max-rocket-internet) [inline stable/listmonk/values.yaml:7]: Can we include a tag?

Comment (javad-hajiani) [reply stable/listmonk/values.yaml:7]: I thought by default it would be better to get image version from Chart.yaml, wdyt?

Comment (max-rocket-internet) [reply stable/listmonk/values.yaml:7]: Oh yes, that's better!

**Review by max-rocket-internet — APPROVED** [review]

File: stable/listmonk/templates/deployment.yaml (hunk 1 above)
Comment (javad-hajiani) [reply stable/listmonk/templates/deployment.yaml]: Fixed, Thanks

**Pull request merged**